### PR TITLE
Add group to pebble layer.

### PR DESCRIPTION
**Pull request by WRFitch**
## Proposal
adds user group when running pgbouncer through pebble

## Testing
- Updated unit test to match.

### CODE REVIEW

File: src/charm.py [hunk 1]
```diff
@@ -120,6 +120,7 @@ def _pgbouncer_layer(self) -> Layer:
                 PGB: {
                     "summary": "pgbouncer service",
                     "user": PG_USER,
+                    "group": PG_USER,
```
Comment (taurus-forever) [inline src/charm.py:123]: LGTM. Nit: I would add constant PG_GROUP to avoid questions and confusion in the future :-)

**Review by taurus-forever — DISMISSED** [review]
LGTM

**Review by marceloneppel — DISMISSED** [review]

File: src/charm.py (hunk 1 above)
Comment (taurus-forever) [reply src/charm.py]: Thank you for addressing it!

**Review by taurus-forever — APPROVED** [review]
LGTM. I like it! (c)

**Review by marceloneppel — DISMISSED** [review]

**Pull request merged**